Lib/test/test_capi.py

```diff
@@ -335,7 +335,7 @@ class C(): pass
             *_, count = line.split(b' ')
             count = int(count)
             self.assertLessEqual(count, i*5)
-            self.assertGreaterEqual(count, i*5-1)
+            self.assertGreaterEqual(count, i*5-2)
```

Comment:
Just curious, what's going on here?

Reply:
We are doing an extra allocation when quickening.
The original test (in 3.10) was `i*5`, then we added an extra allocation for `co_quickened`, so it went down to `i*5-1`, and now it drops to `i*5-2`.